pilot/cmd/pilot-agent/main.go

```diff
@@ -142,6 +149,40 @@ var (
 			role.TrustDomain = spiffe.GetTrustDomain()
 			log.Infof("Proxy role: %#v", role)
 
+			// Add cert paths as node metadata only if they differ from defaults
```

Comment:
Should we always set the metadata? we handle the absence of these fields properly in Pilot-discovery.